### PR TITLE
Added "nanquantile" to Numpy Frontend #19505

### PR DESCRIPTION
**Pull request by anuragnkoppal**
can you please review and merge this

### CODE REVIEW

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

**Comment by zhumakhan** [comment]
close #19505

**Comment by zhumakhan** [comment]
Please use `ivy.nanquantile` like it was done in paddle frontend: https://github.com/unifyai/ivy/blob/master/ivy/functional/frontends/paddle/tensor/stat.py#L30
Also please add tests for this. If you have any question please let me know.
Thanks!

**Comment by anuragnkoppal** [comment]
@zhumakhan added test_numpy_nanquantile please check

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_statistics/test_order_statistics.py [hunk 1]
```diff
@@ -81,3 +81,38 @@ def test_numpy_nanpercentile(
         test_flags=test_flags,
         input_dtypes=input_dtypes,
     )
+
+
+def test_numpy_nanquantile(
```
Comment (zhumakhan) [inline ivy_tests/test_ivy/test_frontends/test_numpy/test_statistics/test_order_statistics.py:86]: Could you please wrap this with proper wrapper functions like in `test_numpy_nanpercentile`?

**Review by zhumakhan — CHANGES_REQUESTED** [review]

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_statistics/test_order_statistics.py (hunk 1 above)
Comment (anuragnkoppal) [reply ivy_tests/test_ivy/test_frontends/test_numpy/test_statistics/test_order_statistics.py:92]: wrapped!

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_statistics/test_order_statistics.py [hunk 2]
```diff
@@ -42,43 +86,32 @@ def test_numpy_ptp(
...
-        where=where,
-        input_dtype=input_dtypes,
-        test_flags=test_flags,
-    )
-
     np_frontend_helpers.test_frontend_function(
-        a=values[0][0],
-        q=values[0][1],
+        a=values[0],
+        q=values[1],  # Using the second value which represents quantiles
```
Comment (zhumakhan) [inline ivy_tests/test_ivy/test_frontends/test_numpy/test_statistics/test_order_statistics.py:110]: I thinks tests is failing due to values being a list with the single element, which makes `values[1]` invalid

**Review by zhumakhan — CHANGES_REQUESTED** [review]

**Pull request closed without merging**